server/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Also make sure do you really need this file in server folder. maybe .babelrc in root folder will be enough